inc/Engine/Optimization/RUCSS/AbstractAPIClient.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Optimization\RUCSS;
+
+abstract class AbstractAPIClient {
+	/**
+	 * API URL.
+	 */
+	const API_URL = 'https://central-saas.wp-rocket.me/';
+
+	/**
+	 * SAAS main API path.
+	 */
+	const TREESHAKE_PATH = 'api';
+
+	/**
+	 * API Warmup path.
+	 */
+	const WARMUP_PATH = 'warmup';
```

Comment:
@crystinutzaa 

what do you think about defining those constants at their APIClients so at treeshaker APIClient we can define this `TREESHAKE_PATH` and on warmup APIClient we can define this `WARMUP_PATH`?

Reply:
yeah...each path can be defined in each sub-class.